fair_mango/metrics/base.py

```diff
@@ -20,7 +20,7 @@ def is_binary(y: pd.Series | pd.DataFrame) -> bool:
     Returns
     -------
     bool
-        true if data contains two unique values else false
+        True if data contains two unique values else False
     """
     try:
```

Comment:
suggestion: you could use isinstance here instead of try/except

Reply:
Yup, it is indeed better and that way we can catch ValueErrors from other sources